tests/plugins/test_pulp_pull.py

```diff
@@ -155,6 +182,9 @@ def custom_get_v2_no_headers(self, url, headers, **kwargs):
     def custom_get_v2_broken(self, url, headers, **kwargs):
         return self.config_response_config_v2_broken
 
+    """
+    """
```

Comment:
what's this?

Reply:
scaffolding from some debug code.

Removed and fixed.